terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -223,7 +223,7 @@ resource "kubernetes_role" "workflow_manager_role" {
 
   rule {
     // API group "" means the core API group.
-    api_groups = [""]
+    api_groups = ["batch", ""]
     // Workflow manager can list pods and create jobs.
     // Note: Some of these permissions will probably wind up not being needed.
     // Starting with a moderately generous demonstration set.
```

Comment:
I see your comment here about narrowing the permissions later, but I wonder if we can have multiple `rule` blocks so we can use separate sets of nouns and verbs with the `batch` and `""` API groups, or if we have to define distinct `kubernetes_roles` to do that.